taxi/forms.py

```diff
@@ -0,0 +1,63 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def validate_license_number(license_number):
+    if not len(license_number) == 8:
+        raise ValidationError("Ensure that length of license equals 8")
+    for char in license_number[:3]:
+        if not char.isalpha():
+            raise ValidationError(
+                "Ensure that first 3 characters are letters"
+            )
+        if not char.isupper():
+            raise ValidationError(
+                "Ensure that first 3 characters are uppercase"
+            )
```

Comment:
there is no need to iterate through each char, `license_number[:3].isalpha()` and so on work in the same way